index.html

```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Conference Page - home</title>
+  <link rel="stylesheet" href="styles.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,400;0,700;0,900;1,400&display=swap" rel="stylesheet">
+</head>
+<body>
+  <header>
+    <nav class="nav-bar">
+      <a class="mobile menu-h "><img class='mobile menu-h' src="Icons/menu.png"></a>
+      <a href="index.html"><div class="nav-logo"></div></a> 
+      <ul class="desktop nav-ul">
+        <li class="nav-il"><a href="about.html">about</a></li>
+        <li class="nav-il"><a href="about.html">Program</a></li>
+        <li class="nav-il"><a href="about.html">Sponsor</a></li>
+        <li class="nav-il"><a href="about.html">News</a></li>
+        <li class="nav-il"><a href="about.html">Stoic campaing</a></li>
+      </ul>
+    </nav>
+  </header>
+  <div id="modal"class="modal mobile">
+    <img class="modal-close" src="Icons/Enabled2.svg">
+    <ul class="ul-modal">
+        <li class="il-modal"><a href="about.html">about</a></li>
+        <li class="il-modal"><a href="about.html">Program</a></li>
+        <li class="il-modal"><a href="about.html">Sponsor</a></li>
+        <li class="il-modal"><a href="about.html">News</a></li>
+        <li class="il-modal"><a href="about.html">Stoic campaing</a></li>
+      </ul>
+  </div>
+  <section id="home-section" class="section">
+    <span id="home-section-span-title" class="home-section-span">In search of the virtue</span>
+    <h1 class="home-section-h1 bold">STOIC LIFESTYLE SUMMER 2023</h1>
+    <div class="home-section-div">
+      <span class="home-section-span">Discover the meaning of life according to the stoics, knowing about their principles, lifestyle, and precepts, from the hand of their most influential authors, taken place in June, in Pereira / COL.</span>
+    </div>
+    <span id="home-section-span-date" class="home-section-span bold">2023.06.21(WED) ~ 25(SUN)</span>
+    <span class="home-section-span2">@ Conventions center "Expofuturo" Pereira.</span>
+  </section>
+  <section id="program-section">
+    <h2 class="h2-title">Main program</h2>
+    <hr class="section-hr">
+    <div id="program-section-container">
+      <div class="program-section-item">
+        <img 
+        class="program-section-img"
+        src="Icons/speaker.png">
+        <h3 
+        class="program-section-h3">
+          Lecture
+        </h3>
+        <p
+        class="program-section-p">
+          Listen to the speakers from various countries about the messeges of sharing and opening.
+        </p>
+      </div>
+      <div class="program-section-item">
```

Comment:
- [ ] I would kindly suggest you add the `See more program` button to your main program page on `desktop view` as seen in the Behance design and also kindly add some `hover` effect to the cards.

Behance design 👇 

![beh2](https://user-images.githubusercontent.com/78634841/217395637-c4fbd37e-0052-4073-89d9-ae09b78cd65d.png)